app/assets/stylesheets/govuk_publishing_components/components/_input.scss

```diff
@@ -49,4 +48,12 @@
     height: govuk-px-to-rem(40px);
   }
   margin-top: 0;
+
+  &.gem-c-input__affix--prefix {
```

Comment:
The prefix and suffix should be separate element classes rather than modifiers.

Reply:
Why? They're basically the same element but for the different border.

Reply:
Hi Simon, I made them modifiers to cut down on the amount of repeated code and to make maintenance easier. I will concede that separate element classes are perhaps simpler though. 

I'm not sure i'm following your point on adding separate modifiers to the prefix and suffix classes. In the case of errors they work on the input component itself rather than the prefix/suffix. 

<img width="792" alt="Screenshot 2020-05-19 at 11 08 08" src="https://user-images.githubusercontent.com/41922771/82314060-26634800-99c1-11ea-8eae-d46751d863dd.png">

Is there a scenario you can envision in which we may need to add a modifier that is not applicable to both the prefix and suffix?